crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -88,6 +88,12 @@ pub struct Circle {
 impl Primitive2d for Circle {}
 
 impl Circle {
+    /// Create a new [`Circle`] from a `radius`
```

Comment:
```suggestion
    /// Create a new [`Circle`] from a `radius`.
```

Reply:
Most constructors (`Plane2d::new`, `Segment2d::new`, `Polyline2d::new`, `Triangle2d::new`, `Rectangle::new`...) in these files currently don't end the doc comment with a period unless the comment spans multiple lines. I'd like to change them all to end with a period and be more consistent though.